### PR TITLE
Create navbar

### LINKED ISSUE #13: Userstory #7: Create NavBar
## Value proposition

---

**As a** user
**I want to**  switch quickly through the pages
**so that** I don't have to navigate back to the home- page to switch the category.

## Description

---

Image

## Acceptance criteria

---

- [ ] the categories serve as navigation-items
- [ ] the user can see the navigation-bar at any time except on the homepage
- [ ] the navigation items have the same icons as the navigation on the "home"
- [ ] when the user clicks on the navigation it leads them to the different pages: "Schlafen", "Hygiene", "Medizin": here are the facilities displayed which belong in this category
- [ ] the user can navigate through the pages
- [ ] each page is reachable through the navigation bar
- [ ] the page the user currently visits is marked in the navigation-bar with a different color
- [ ] every page has a navigation on the bottom (except homepage)
- [ ]  every page has its own header at the top with the headline of the page
- [ ] every page has a head with a title which is displayed in the tab

## Tasks

---

- [x] open feature branch
- [x] reuse `Home-Link` and `Favorite-Link` for NavBar
- [x] delete one of those components and rename it as NavBar
- [x] add `Link` from next/link for  Schlafen, Hygiene, Medizin to `Home- `and `Favorite- Link`
- [x] add icons to NavBar
- [x] add `NavBar-Component` to each page: sleep, medicine and hygiene

    - [x] create Layout-Component import Header
    - [x] update Header to display matching heading from each page
    - [x] add NavBar to Layout as footer
    - [x]  implement Layout in app.js
    - [x] hide the NavBar on landing-page
    - [x] create Head- Component
    - [x] each Head should have a <title> and a tag
    - [x]  add Head to each page
    - [x] remove previous headlines
  

## Complexity Estimation

---

M

### PR DESCRIPTION
**Pull request by BucherLara**

### CODE REVIEW

File: components/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          </svg>
+        </StyledLink>
+      </StyledNav>
+    </StyledFooter>
+  );
+}
+
+const StyledLink = styled(Link)`
+  // display: flex;
+  // width: 100%;
```
Comment (stefankorder) [inline components/Footer/index.js]: If you don't need code anymore, please delete the unused lines.

File: components/Footer/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  padding: 0;
+  ${({ active }) => (active ? "fill: #DAF7A6" : "fill:black")}
+`;
+const StyledFooter = styled.footer`
+  // bottom: 100px;
+  // position: fixed;
```
Comment (stefankorder) [inline components/Footer/index.js]: If you don't need code anymore, please delete the unused lines.

**Review by stefankorder — APPROVED** [review]
LGTM! Please take a look at my comments.

**Review by HendrikNeuschulz — APPROVED** [review]
Nice Job !

**Pull request merged**